packages/vite/src/client/overlay.ts

```diff
@@ -113,6 +113,8 @@ pre.frame {
 .stack {
   font-size: 13px;
   color: var(--dim);
+  white-space: pre-wrap;
+  word-break: break-all;
```

Comment:
Maybe the style you modified can be applied to the mobile version independently.

Reply:
> Maybe the style you modified can be applied to the mobile version independently.

You mean `media query` ?

Reply:
Yes.

Reply:
@btea need review

Reply:
Looks good to me, but I'm not sure whether the specific value of padding needs to be adjusted. Looking forward to others reviewing it.

Reply:
@sapphi-red could U please review for this ?